cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition/help-script.html

```diff
@@ -23,3 +23,39 @@
     but if you specify multiple parameters they must all be named:
 </p>
 <pre>stage name: 'Build', concurrency: 1</pre>
+
+<h3>Environment Variables</h3>
+
+A set of environment variables are made available to all Jenkins Job types, including Workflow Jobs.
+The following is a general list of variables (by name) that are available to all Job types. See the notes
+below the list for Workflow specific details.
+
+<p>
+    <iframe src="${rootURL}/env-vars.html" width="100%"></iframe>
```

Comment:
Did you test this—does `${rootURL}` actually work here? AFAIK this does not work for a help page rendered as static HTML, which is why the standard link comes from `Shell/config.groovy` in `entry.description`, rather than from a help page. The one example I know of where dynamic content is rendered in a help page is `ArtifactArchiver/help-defaultExcludes.groovy`.